content/en/blog/_posts/2022-11-11-retroactive-default-storage-class.md

```diff
@@ -0,0 +1,163 @@
+---
+layout: blog
+title: "Kubernetes 1.26: Retroactive Default StorageClass"
+date: 2022-12-08
```

Comment:
```suggestion
date: 2023-01-05
```
and please update the path `content/en/blog/_posts/2022-11-11-retroactive-default-storage-class.md` to match.